packages/NodeTypeResolver/Node/AttributeKey.php

```diff
@@ -4,8 +4,6 @@
 
 namespace Rector\NodeTypeResolver\Node;
 
-use PHPStan\Analyser\Scope;
```

Comment:
hm.., it seems cause it used as `@see Scope` removed

Reply:
I see, it because `@see` is exists inside `PHPStan\PhpDocParser\Ast\PhpDoc\PhpDocTextNode`:

```
PHPStan\PhpDocParser\Ast\PhpDoc\PhpDocTextNode #18819
   text: 'Contains @see Scope'
```

Reply:
Fixed by make `@see` not part of text, but independent docblock https://github.com/rectorphp/rector-src/pull/5012/commits/ac93c3e0599f48e263bc1a693419ae092add00ff